data/avro-schema/src/main/resources/avro/net/corda/data/persistence/EntityResponseSuccess.avsc

```diff
@@ -1,16 +1,16 @@
 {
   "type": "record",
   "name": "EntityResponseSuccess",
-  "doc": "An {@link EntityRequest} has been successful.  A return type of `null` can be considered success and/or no results.  See also:  {@link EntityResponseFailure} and {@link EntityResponse}",
+  "doc": "An {@link EntityRequest} has been successful.  See also:  {@link EntityResponseFailure} and {@link EntityResponse}",
   "namespace": "net.corda.data.persistence",
   "fields": [
     {
       "name": "result",
```

Comment:
I think this should be `results` to be consistent.

Reply:
Done